common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -165,4 +165,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2018, 9, 6),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-acquisitions-epic-optimize-a-a-test",
```

Comment:
Should probably be used to enable/disable the test

Reply:
yes, it will be thanks to `test-can-run-checks.js` as per normal A/B test functionality - unless I've misunderstood?

Reply:
ah no, it's been a long time since I looked at ab tests 🙈